test/elixir_internal_certificate/account/accounts_test.exs

```diff
@@ -0,0 +1,181 @@
+defmodule ElixirInternalCertificate.Account.AccountsTest do
+  use ElixirInternalCertificate.DataCase
```

Comment:
Should we run this test as async mode @nvminhtue you can check it on https://hexdocs.pm/ex_unit/1.13/ExUnit.Case.html

![image](https://user-images.githubusercontent.com/11751745/173299327-6c883436-e2ed-49cd-8bb5-e4a21661e5f6.png)